src/styles/common/_buttons.scss

```diff
@@ -121,7 +121,8 @@ button svg {
     &:not(.bdl-is-disabled),
     &:not(.is-disabled) {
         &:focus {
-            border-color: $primary-color;
+            background-color: lighten($primary-color, 8%);
+            border: 2px solid $primary-color;
```

Comment:
If the border width changes on focus, won't it cause layout shifts?

Reply:
I'll switch it back to 1px :D

Reply:
Even with 1px, the default is 0, so it will still shift.